src/modules/Popup/Popup.js

```diff
@@ -0,0 +1,196 @@
+import React, { Component, PropTypes } from 'react'
+import { findDOMNode } from 'react-dom'
+import { META } from '../../lib'
+import Portal from '../../addons/Portal/Portal'
+import cx from 'classnames'
+import isString from 'lodash/isString'
```

Comment:
Great, I got into the habit of cherry picking method of the any library for size concerns. I was wondering if the same applies to stardust element then? If someone only wanted to import the Popup functionality, using: 

```jsx
import Portal from '../../addons/Portal/Portal'
```

instead of 

```jsx
import { Portal } from '../../addons'
```

in the Popup wouldn't impact the size of the import by loading the whole `addons` content ?